src/app/website/posts/zkp.markdown

```diff
@@ -0,0 +1,74 @@
+---
+title: Zero-knowledge Proofs&#58; An Intuitive Explanation
+date: 2019-05-13
+author: Vanishree Rao
+---
+
+If you are hearing about zero-knowledge proofs -- a fascinating cryptography tool, for the first time, you are likely to say "Hah! That sounds impossible." Read on to get an intuitive understanding of what they are. But first, some background. Zero-knowledge proofs (or ZKPs)  were [invented](http://people.csail.mit.edu/silvio/Selected%20Scientific%20Papers/Proof%20Systems/The_Knowledge_Complexity_Of_Interactive_Proof_Systems.pdf) by Shafi Goldwasser, Silvio Micali, and Charles Rackoff in 1985. Ever since, ZKPs have been one of the most active areas of research in Cryptography. Moreover, recently, they are enjoying significant impact on real-world applications, specifically on blockchain technologies. [Zcash](https://z.cash/technology/), a pioneering blockchain project, employed ZKPs to achieve anonymity in financial transactions. At O(1)Labs, we are building CODA, the first succinct blockchain, using ZKPs. No matter how many transactions are recorded on the blockchain, the blockchain remains at most the size of a few tweets. 
```

Comment:
Maybe you could add one sentence before the first one where you introduce zero-knowledge proofs via anecdote to lead into the "if you are hearing about zero-knowledge proofs for the first time..."

Reply:
Done. Added "Zero-knowledge proofs are fundamental cryptography primitive. "  What do you think? Is the grammar correct?